app/forms/hyrax/forms/permission_template_form.rb

```diff
@@ -73,8 +69,8 @@ def update(attributes)
       # Copy this access to the permissions of the Admin Set or Collection and to
       # the WorkflowResponsibilities of the active workflow if this is an Admin Set
       def update_access(remove_agent: false)
-        source_model.reset_access_controls!
-        update_workflow_responsibilities(remove_agent: remove_agent) if source_model.is_a?(AdminSet)
+        model.reset_access_controls_for(collection: source)
```

Comment:
This is part of the fix of the issue.  `source_model` uses `ActiveFedora::Base.find` which is not available when using Valkyrie postgres adapter.  `source` uses `Hyrax.query_service.find_by`.